doc/glossary.rst

```diff
@@ -76,13 +76,30 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward solution
+        The forward solution (abbr. ``fwd``) is a linear operator capturing the
+        relationship between each dipole location in the :term:`source space`
+        and the corresponding field distribution measured by the sensors (AKA,
+        the "lead field matrix"). Calculating a forward solution requires a
+        conductivity model of the head, encapsulating the geometry and
+        electrical conductivity of the different tissue compartments (see
+        :term:`boundary element model <BEM>` and
+        :class:`mne.bem.ConductorModel`).
+
     info
         Also called ``measurement info``, it is a collection of metadata regarding
         a Raw, Epochs or Evoked object; e.g.,
         channel locations and types, sampling frequency,
         preprocessing history such as filters ...
         See :ref:`tut-info-class` for a narrative overview.
 
+    inverse operator
+        The inverse operator is an :math:`M \times N` matrix (:math:`M` source
+        locations by :math:`N` sensors) that, when applied to the sensor
+        signals, yields estimates of the brain activity that gave rise to the
+        observed sensor signals. Inverse operators are available for the linear
+        inverse methods MNE, dSPM, sLORETA and eLORETA.
```

Comment:
shall we point to `plot_compute_mne_inverse_volume` and others?

Reply:
I'm punting this until a later PR where the whole glossary gets better cross-referencing.